src/open_inwoner/scss/components/Contactmomenten/Contactmomenten.scss

```diff
@@ -74,46 +74,22 @@
 }
 
 .contactmoment {
-  &__details {
-    // Set table width for word-break
-    table {
-      margin-top: 0;
-      overflow-x: auto;
-      table-layout: fixed;
-      width: 100%;
-      @media (min-width: 768px) {
-        width: var(--mobile-ms-width);
-      }
-    }
-    table th {
-      width: 100px;
-      white-space: nowrap;
-    }
-    table td {
-      width: 300px;
-      overflow: hidden;
-      word-break: break-word;
-      white-space: normal;
-
-      &.table__item--status {
-        .table__item--wrapper {
-          display: flex;
-          justify-content: space-between;
-        }
-      }
+  &__heading {
+    font-size: var(--font-size-body-large);
```

Comment:
This is not the correct CSS variable, and you have not applied the correct Font-family for Headings and the other needed values. 
We already have these Heading sizes, but you - you would need --font-size-heading-3; here (which is absolutely fine to use for a Level 2 heading).
You can just inherit everything from Heading 3:

```color: var(--font-color-heading-3);
  font-family: var(--font-family-heading);
  font-size: var(--font-size-heading-3);
  font-weight: var(--font-weight-heading-3);
  line-height: var(--font-line-height-heading-3);
  margin: 0;
```